src/main/java/Parser.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Parser class interprets the user's command.
+ */
+public class Parser {
+
+    /**
+     * Parses the input command given by the user to determine which functionality to execute.
+     *
+     * @param input String entered by the user, denoting instructions for the program.
+     */
+    public static void parseInput(String input) {
+        String[] inputParts = input.split(" ");
+        String instruction = inputParts[0];
+
+        switch (instruction) {
+        case "done":
```

Comment:
Hmmm... here you could consider using command word as constants instead of "magic literals"